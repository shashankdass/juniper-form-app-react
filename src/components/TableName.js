import React, { Component } from 'react';
import { Col, FormGroup, Label, Input } from 'reactstrap';

export class TableName extends Component{

  render() {
    return(
      <div>
      <FormGroup row>
        <Label sm={2}>
          Table Name:
          </Label>
          <Col sm={5}>
            <Input name="tableName" type="text"  />
          </Col>
        </FormGroup >
      </div>
    )
  }
}

export default TableName;
