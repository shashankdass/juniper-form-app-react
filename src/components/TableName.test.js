import React from 'react';
import {TableName} from './TableName';
import renderer from 'react-test-renderer';

test('it renders TableName Component', () => {
  const component = renderer.create(
    <TableName></TableName>
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
