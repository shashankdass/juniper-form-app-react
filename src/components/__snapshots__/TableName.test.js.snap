exports[`test it renders TableName Component 1`] = `
<div>
  <div
    className="row form-group">
    <label
      className="col-sm-2 col-form-label"
      htmlFor={undefined}>
      Table Name:
    </label>
    <div
      className="col col-sm-5">
      <input
        className="form-control"
        name="tableName"
        type="text" />
    </div>
  </div>
</div>
`;
